app/coffee/Notifications.coffee

```diff
@@ -22,25 +22,38 @@ module.exports =
 				logger.log number:number, user_id:user_id, key:notification.key, "alredy has notification key for user"
 				callback number
 			else
-				doc = 
+				doc =
 					user_id: ObjectId(user_id)
 					key: notification.key
 					messageOpts: notification.messageOpts
 					templateKey: notification.templateKey
+					expires: notification.expires || false
+				# ttl index on `expiresFrom` field
+				# in Mongo, TTL indexes only work on date fields,
+				# and ignore the document when that field is missing
+				if doc.expires
+					doc.expiresFrom = new Date()
```

Comment:
what if we did

if notification.expires?
    doc.expires = notification.expires

rather than having expires set to false